drv/stm32h7-update-api/src/lib.rs

```diff
@@ -26,4 +28,35 @@ pub const BLOCK_SIZE_BYTES: usize = FLASH_WORD_BYTES * FLASH_WORDS_PER_BLOCK;
 
 pub const BLOCK_SIZE_WORDS: usize = BLOCK_SIZE_BYTES / 4;
 
+#[derive(
+    Clone,
+    Copy,
+    Eq,
+    PartialEq,
+    FromPrimitive,
+    Serialize,
+    Deserialize,
+    SerializedSize,
+)]
+pub enum SlotId {
+    Active = 0,
```

Comment:
**question:** From the peanut gallery: Does the naming of the slot IDs come from an outside source, or is it otherwise well understood within Oxide?

From an outsider perspective, it seems weird that apparently there are two banks that you can boot to, so either can be the "actively booted bank" but one is still called "inactive" and the other is "active", regardless of which one is actually currently booted into.

I guess these aren't actually slot/bank IDs but rather just a value that can be set as 1 to say "after restart, switch from current bank to other bank"? If that is the case, would this rather be a "pending boot slot value"?